### PR TITLE
[Dashboard] Remove <Stack/> (5)

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on refactoring the code to replace the use of `Stack` components from `@chakra-ui/react` with `div` elements styled with Tailwind CSS classes, enhancing layout consistency and potentially improving performance.

### Detailed summary
- Removed instances of `Stack` in favor of `div` with Tailwind CSS classes.
- Updated various components to use `className` for styling instead of Chakra's `Stack` properties.
- Adjusted layout properties in components for better responsiveness.
- Introduced `StackProps` in `SelectOptionProps` interface.
- Improved accessibility and semantic structure of the components.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4844?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4844** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4844?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4843** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4843?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/contract-components/contract-publish-form/landing-fieldset.tsx [hunk 1]
```diff
@@ -281,7 +281,7 @@ export const LandingFieldset: React.FC<LandingFieldsetProps> = ({
             description="Users will directly deploy the full contract."
             onClick={() => form.setValue("deployType", "standard")}
             isActive={form.watch("deployType") === "standard"}
-            width="full"
+            extraClass="w-full"
```
Comment (jnsdls) [inline apps/dashboard/src/components/contract-components/contract-publish-form/landing-fieldset.tsx:284]: can this just be `className`?

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/core-ui/batch-upload/lazy-mint-form/select-option.tsx [hunk 2]
```diff
@@ -37,17 +40,18 @@ export const SelectOption: React.FC<SelectOptionProps> = ({
...
       shouldWrapChildren
     >
-      <Stack
-        as={Card}
-        padding={5}
+      <Card
+        className={cn(
+          "flex flex-col gap-2 rounded-md p-5",
+          disabled ? "pointer-events-none" : "",
+          disabled ? "cursor-not-allowed" : "cursor-pointer",
```
Comment (jnsdls) [inline apps/dashboard/src/core-ui/batch-upload/lazy-mint-form/select-option.tsx:47]: combine these, it's the same condition

File: apps/dashboard/src/core-ui/batch-upload/lazy-mint-form/select-option.tsx [hunk 3]
```diff
@@ -37,17 +40,18 @@ export const SelectOption: React.FC<SelectOptionProps> = ({
...
+          className || "",
```
Comment (jnsdls) [inline apps/dashboard/src/core-ui/batch-upload/lazy-mint-form/select-option.tsx:48]: no need for `|| ""`

**Review by jnsdls — CHANGES_REQUESTED** [review]

**Review by jnsdls — APPROVED** [review]

**Pull request merged**